pages/en/deploying/deploying-a-subgraph-to-hosted.mdx

```diff
@@ -2,49 +2,11 @@
 title: Deploying a Subgraph to the Hosted Service
 ---
 
-To see instructions on how to deploy your subgraph to the Hosted Service, [click here](#create-a-hosted-service-account).
```

Comment:
I removed this sentence, as it simply links one paragraph down on the same page. 
After moving "supported networks" it will link one sentence down

Reply:
Good call, thank you!